docs/en/sql-reference/functions/other-functions.md

```diff
@@ -571,7 +571,7 @@ Example:
 
 ``` sql
 SELECT
-    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vk.com'], ['www.yandex', 'example.com']) AS s,
+    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vkontakte.ru'], ['www.yandex', 'example.com', 'vk.com'] AS s,
```

Comment:
you lost closing bracket of `transform()`

Reply:
Omfg...
Will fix now.

Reply:
Thanks.
My first web GITing experience... :)